pulumi/infra/consul_intentions.py

```diff
@@ -36,5 +36,7 @@ def __init__(
                         kind=intention["Kind"],
                         name=intention["Name"],
                         config_json=json.dumps({"Sources": intention["Sources"]}),
-                        opts=pulumi.ResourceOptions(parent=self),
```

Comment:
So because we're setting the opts here, consul uses the default address (localhost:8500).